SONAR-netCDF4/docs/tableBeamGroup1.adoc

```diff
@@ -158,4 +158,8 @@ transmitted pulse" | |
 | | |
 |transmit_t transmit_type(ping_time) |M |Type of transmit pulse.
 |:long_name = "Type of transmitted pulse" | |
+| | |
+|int beam_rx_transducer_index(ping_time, beam) |MA |receiving or monostatic transducer index associated with the given beam
+|:valid_min = "0" | |
+|:long_name = "receiving transducer index associated with the given beam" | |
```

Comment:
Tried to be have a clearer long name. Not sure if I succeeded...
```suggestion
|:long_name = "Index into the transducer vector variables to indicate transducer function" | |
```

Reply:
Not convinced by this one, it does not indicate the transducer function (we can have several transducer with the same function) but the transducer itself.

Reply:
Not sure either, everytime it is paraphrasing the comment of the variable ...

```suggestion
|:long_name = "Index into the transducer vector variables linking a beam to the transducer" | |
```

Reply:
Good point about the paraphrasing of the comment. The netCDF/CF guidelines for long_name is something that is useful for a plot label. With that guidance, how about simply "Receive transducer index"?

Reply:
Good for me, we use long names in our applications to display axis selection and if it is too long it's become an issue for us.
I will commit your suggestion

```suggestion
|:long_name = "Receive transducer index" | |
```